taxi/views.py

```diff
@@ -40,15 +42,60 @@ class CarListView(LoginRequiredMixin, generic.ListView):
     queryset = Car.objects.all().select_related("manufacturer")
 
 
-class CarDetailView(LoginRequiredMixin, generic.DetailView):
+class CarDetailView(LoginRequiredMixin, DetailView):
+    model = Car
+    template_name = "taxi/car_detail.html"
```

Comment:
You don't need to override the context_object_name if it is the same as the default one. In this case, the default name is 'car', which is the same as the one you've set. So you can safely remove this line.